File: SourceCode/Debug Helpers/open.fda.gov/ApiDalc/DataObjects/SearchSummary.vb
Imports ApiDalc.Enumerations

Namespace DataObjects

    Public Class SearchSummary

        Public Property Keyword As String

        Public Property EventCount As Integer

        Public Property ClassICount As Integer
        Public Property ClassIDescription As String = GetEnumDescription(Classification.Class_I)

        Public Property ClassIICount As Integer
        Public Property ClassIIDescription As String = GetEnumDescription(Classification.Class_II)

        Public Property ClassIIICount As Integer
        Public Property ClassIIIDescription As String = GetEnumDescription(Classification.Class_III)

    End Class

End Namespace

File: SourceCode/AspCoreVersion/src/ShopAware.Core/DataObjects/SearchSummary.cs
namespace ShopAware.Core
{
    namespace DataObjects
    {
        public class SearchSummary
        {
            #region Properties

            public string Keyword { get; set; }

            public string State { get; set; }

            public int EventCount { get; set; }

            public int ClassICount { get; set; }

            //  Public Property ClassIDescription As String = GetEnumDescription(Classification.Class_I)

            public int ClassIICount { get; set; }

            //  Public Property ClassIIDescription As String = GetEnumDescription(Classification.Class_II)

            public int ClassIIICount { get; set; }

            #endregion
        }
    }
}